[Bug]: cannot find module register.js

### What happened?

A few hours ago when pulling latest docker image thru Coolify, it started being unable to start the container with the following error message and doing an infinite restart loop:

```
node:internal/modules/esm/resolve:275
    throw new ERR_MODULE_NOT_FOUND(
          ^

Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/app/scripts/register.js' imported from /app/
    at finalizeResolution (node:internal/modules/esm/resolve:275:11)
    at moduleResolve (node:internal/modules/esm/resolve:860:10)
    at defaultResolve (node:internal/modules/esm/resolve:984:11)
    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:780:12)
    at #cachedDefaultResolve (node:internal/modules/esm/loader:704:25)
    at ModuleLoader.resolve (node:internal/modules/esm/loader:687:38)
    at ModuleLoader.getModuleJobForImport (node:internal/modules/esm/loader:305:38)
    at onImport.tracePromise.__proto__ (node:internal/modules/esm/loader:643:36)
    at TracingChannel.tracePromise (node:diagnostics_channel:344:14)
    at ModuleLoader.import (node:internal/modules/esm/loader:642:21) {
  code: 'ERR_MODULE_NOT_FOUND',
  url: 'file:///app/scripts/register.js'
}

Node.js v22.16.0
```

### Distribution version

Linux

### App Version

_No response_

### What browsers are you seeing the problem on if you're using web version?

_No response_

### Are you self-hosting?

- [x] Yes

### Self-hosting Version

ghcr.io/toeverything/affine-graphql:stable

### Relevant log output

```shell
node:internal/modules/esm/resolve:275
    throw new ERR_MODULE_NOT_FOUND(
          ^

Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/app/scripts/register.js' imported from /app/
    at finalizeResolution (node:internal/modules/esm/resolve:275:11)
    at moduleResolve (node:internal/modules/esm/resolve:860:10)
    at defaultResolve (node:internal/modules/esm/resolve:984:11)
    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:780:12)
    at #cachedDefaultResolve (node:internal/modules/esm/loader:704:25)
    at ModuleLoader.resolve (node:internal/modules/esm/loader:687:38)
    at ModuleLoader.getModuleJobForImport (node:internal/modules/esm/loader:305:38)
    at onImport.tracePromise.__proto__ (node:internal/modules/esm/loader:643:36)
    at TracingChannel.tracePromise (node:diagnostics_channel:344:14)
    at ModuleLoader.import (node:internal/modules/esm/loader:642:21) {
  code: 'ERR_MODULE_NOT_FOUND',
  url: 'file:///app/scripts/register.js'
}

Node.js v22.16.0
```

### Anything else?

_No response_

@fengmk2 

Tested all combinations with and without `NODE_OPTIONS=--import=./scripts/register.js`, `AFFINE_INDEXER_ENABLED=false` using both pgvector:pg16 and postgres:16-alpine.
With the same compose setup all combinations:
stable doesn't works.
stable-e043ecf works well.

## docker-compose:
```yaml
x-default-healthcheck: &default-healthcheck
  interval: 60s
  timeout: 20s
  retries: 5
  start_period: 60s
  start_interval: 5s
services:
  affine:
    image: ghcr.io/toeverything/affine-graphql:stable
    container_name: affine
    restart: unless-stopped
    environment:
      - REDIS_SERVER_HOST=affine-redis
      - DATABASE_URL=postgresql://affine-user:changeme@affine-db:5432/affine # Change
      #- 'NODE_OPTIONS=--import=./scripts/register.js'
      - AFFINE_SERVER_HTTPS=true
      - 'AFFINE_SERVER_EXTERNAL_URL=https://affine.site.com' # Change
      - 'AFFINE_SERVER_HOST=https://affine.site.com' # Change
      - 'AFFINE_CONFIG_PATH=/root/.affine/config'
      - NODE_ENV=production
      - MAILER_HOST=smtp.site.com # Change
      - MAILER_PORT=465 # Change
      - MAILER_USER=admin@site.com.com # Change
      - MAILER_SECURE=true
      - MAILER_PASSWORD=mailpass # Change
      - MAILER_SENDER=admin@site.com # Change
    expose:
      - 3010
    volumes:
      - ./storage:/root/.affine/storage
      - ./config:/root/.affine/config
    networks:
      - affine-network
      - proxy
    depends_on:
      affine-redis:
        condition: service_healthy
      affine-db:
        condition: service_healthy
      affine_migrations:
        condition: service_completed_successfully
    healthcheck:
      test: ["CMD", "echo", "health"]
      <<: *default-healthcheck
  affine_migrations:
    image: ghcr.io/toeverything/affine-graphql:stable
    container_name: affine_migrations
    restart: on-failure
    volumes:
      - ./storage:/root/.affine/storage
      - ./config:/root/.affine/config
    command: ['sh', '-c', 'node ./scripts/self-host-predeploy.js']
    environment:
      - REDIS_SERVER_HOST=affine-redis
      - DATABASE_URL=postgresql://affine-user:changeme@affine-db:5432/affine # Change
      #- 'NODE_OPTIONS=--import=./scripts/register.js'
      - AFFINE_SERVER_HTTPS=true
      - 'AFFINE_SERVER_EXTERNAL_URL=https://affine.site.com' # Change
      - 'AFFINE_SERVER_HOST=https://affine.site.com' # Change
      - 'AFFINE_CONFIG_PATH=/root/.affine/config'
      - 'AFFINE_UPLOAD_PATH=/root/.affine/storage'
      - NODE_ENV=production
    networks:
      - affine-network
    depends_on:
      affine-db:
        condition: service_healthy
      affine-redis:
        condition: service_healthy
  affine-redis:
    image: redis:latest
    restart: unless-stopped
    volumes:
      - affine-redis:/data
    networks:
      - affine-network
    healthcheck:
      test: ["CMD", "redis-cli", "ping"]
      <<: *default-healthcheck
  affine-db:
    image: pgvector/pgvector:pg16
    restart: unless-stopped
    environment:
      - POSTGRES_DB=affine
      - POSTGRES_PASSWORD=changeme # Change
      - POSTGRES_USER=affine-user
      - POSTGRES_PORT=5432
      - POSTGRES_INITDB_ARGS=--data-checksums
      - POSTGRES_HOST_AUTH_METHOD=trust
    volumes:
      - "affine-pg:/var/lib/postgresql/data"
    networks:
      - affine-network
    healthcheck:
      test: ["CMD-SHELL", "pg_isready -U $${POSTGRES_USER} -d $${POSTGRES_DB}"]
      <<: *default-healthcheck
networks:
  affine-network:
    name: affine-network
    driver: bridge
  proxy:
    external: true
volumes:
  affine-pg:
  affine-redis:

```

## `pgvector/pgvector:pg16` error :
```
No pending migrations to apply.
$ cross-env SERVER_FLAVOR=script node ./dist/main.js run
[Nest] 187  - 06/05/2025, 1:45:49 AM     LOG [NestFactory] Starting Nest application...
[Nest] 187  - 06/05/2025, 1:45:49 AM     LOG [InstanceLoader] CommandRootModule dependencies initialized +57ms
[Nest] 187  - 06/05/2025, 1:45:49 AM     LOG [InstanceLoader] ClsModule dependencies initialized +1ms
[Nest] 187  - 06/05/2025, 1:45:49 AM     LOG [InstanceLoader] ClsCommonModule dependencies initialized +0ms
[Nest] 187  - 06/05/2025, 1:45:49 AM     LOG [InstanceLoader] StorageProviderModule dependencies initialized +0ms
[Nest] 187  - 06/05/2025, 1:45:49 AM     LOG [InstanceLoader] ErrorModule dependencies initialized +0ms
[Nest] 187  - 06/05/2025, 1:45:49 AM   DEBUG [ClsModule] ClsInterceptor will be automatically mounted
[Nest] 187  - 06/05/2025, 1:45:49 AM     LOG [InstanceLoader] DiscoveryModule dependencies initialized +15ms
[Nest] 187  - 06/05/2025, 1:45:49 AM     LOG [InstanceLoader] DiscoveryModule dependencies initialized +1ms
[Nest] 187  - 06/05/2025, 1:45:49 AM     LOG [InstanceLoader] ClsRootModule dependencies initialized +1ms
[Nest] 187  - 06/05/2025, 1:45:49 AM     LOG [InstanceLoader] LoggerModule dependencies initialized +1ms
[Nest] 187  - 06/05/2025, 1:45:49 AM     LOG [InstanceLoader] PermissionModule dependencies initialized +1ms
[Nest] 187  - 06/05/2025, 1:45:49 AM     LOG [InstanceLoader] ScannerModule dependencies initialized +0ms
[Nest] 187  - 06/05/2025, 1:45:49 AM     LOG [InstanceLoader] ClsPluginsModule dependencies initialized +0ms
[Nest] 187  - 06/05/2025, 1:45:49 AM     LOG [InstanceLoader] ConfigModule dependencies initialized +1ms
[Nest] 187  - 06/05/2025, 1:45:49 AM     LOG [InstanceLoader] CommandRunnerModule dependencies initialized +0ms
[Nest] 187  - 06/05/2025, 1:45:49 AM     LOG [InstanceLoader] BullModule dependencies initialized +11ms
[Nest] 187  - 06/05/2025, 1:45:49 AM     LOG [InstanceLoader] RedisModule dependencies initialized +1ms
[Nest] 187  - 06/05/2025, 1:45:49 AM     LOG [InstanceLoader] MetricsModule dependencies initialized +0ms
[Nest] 187  - 06/05/2025, 1:45:49 AM     LOG [InstanceLoader] HelpersModule dependencies initialized +0ms
[Nest] 187  - 06/05/2025, 1:45:49 AM     LOG [InstanceLoader] WebSocketModule dependencies initialized +0ms
[Nest] 187  - 06/05/2025, 1:45:49 AM     LOG [InstanceLoader] EventModule dependencies initialized +1ms
[Nest] 187  - 06/05/2025, 1:45:49 AM     LOG [InstanceLoader] PrismaModule dependencies initialized +1ms
[Nest] 187  - 06/05/2025, 1:45:49 AM     LOG [InstanceLoader] BullModule dependencies initialized +2ms
[Nest] 187  - 06/05/2025, 1:45:49 AM     LOG [InstanceLoader] JobModule dependencies initialized +1ms
[Nest] 187  - 06/05/2025, 1:45:49 AM     LOG [InstanceLoader] ServerConfigModule dependencies initialized +1ms
[Nest] 187  - 06/05/2025, 1:45:49 AM     LOG [InstanceLoader] ThrottlerModule dependencies initialized +0ms
[Nest] 187  - 06/05/2025, 1:45:49 AM     LOG [InstanceLoader] CliAppModule dependencies initialized +1ms
[Nest] 187  - 06/05/2025, 1:45:49 AM     LOG [InstanceLoader] CacheModule dependencies initialized +0ms
[Nest] 187  - 06/05/2025, 1:45:49 AM     LOG [InstanceLoader] MutexModule dependencies initialized +0ms
[Nest] 187  - 06/05/2025, 1:45:49 AM     LOG [InstanceLoader] RateLimiterModule dependencies initialized +2ms
[Nest] 187  - 06/05/2025, 1:45:49 AM     LOG [InstanceLoader] ClsPluginModule dependencies initialized +2ms
[Nest] 187  - 06/05/2025, 1:45:49 AM     LOG [InstanceLoader] BullModule dependencies initialized +1ms
[Nest] 187  - 06/05/2025, 1:45:49 AM     LOG [InstanceLoader] IndexerModule dependencies initialized +1ms
[Nest] 187  - 06/05/2025, 1:45:49 AM     LOG [InstanceLoader] ModelsModule dependencies initialized +0ms
[Nest] 187  - 06/05/2025, 1:45:49 AM     LOG [EventBus] Event handler registered [config.init] (OpentelemetryProvider.init)
[Nest] 187  - 06/05/2025, 1:45:49 AM     LOG [EventBus] Event handler registered [config.changed] (OpentelemetryProvider.onConfigChanged)
[Nest] 187  - 06/05/2025, 1:45:49 AM     LOG [EventBus] Event handler registered [config.init] (URLHelper.init)
[Nest] 187  - 06/05/2025, 1:45:49 AM     LOG [EventBus] Event handler registered [config.changed] (URLHelper.init)
[Nest] 187  - 06/05/2025, 1:45:49 AM     LOG [EventBus] Event handler registered [config.init] (CryptoHelper.onConfigInit)
[Nest] 187  - 06/05/2025, 1:45:49 AM     LOG [EventBus] Event handler registered [config.changed] (CryptoHelper.onConfigChanged)
[Nest] 187  - 06/05/2025, 1:45:49 AM     LOG [EventBus] Event handler registered [config.init] (JobExecutor.onConfigInit)
[Nest] 187  - 06/05/2025, 1:45:49 AM     LOG [EventBus] Event handler registered [config.changed] (JobExecutor.onConfigChanged)
[Nest] 187  - 06/05/2025, 1:45:49 AM     LOG [EventBus] Event handler registered [doc.updated] (IndexerEvent.indexDoc)
[Nest] 187  - 06/05/2025, 1:45:49 AM     LOG [EventBus] Event handler registered [workspace.updated] (IndexerEvent.indexWorkspace)
[Nest] 187  - 06/05/2025, 1:45:49 AM     LOG [EventBus] Event handler registered [user.deleted] (IndexerEvent.deleteUserWorkspaces)
[Nest] 187  - 06/05/2025, 1:45:49 AM     LOG [EventBus] Event handler registered [config.init] (ManticoresearchProvider.onConfigInit)
[Nest] 187  - 06/05/2025, 1:45:49 AM     LOG [EventBus] Event handler registered [config.changed] (ManticoresearchProvider.onConfigUpdated)
[Nest] 187  - 06/05/2025, 1:45:49 AM     LOG [EventBus] Event handler registered [config.init] (ElasticsearchProvider.onConfigInit)
[Nest] 187  - 06/05/2025, 1:45:49 AM     LOG [EventBus] Event handler registered [config.changed] (ElasticsearchProvider.onConfigUpdated)
[Nest] 187  - 06/05/2025, 1:45:49 AM     LOG [EventBus] Event handler registered [config.changed.broadcast] (ServerService.onConfigChangedBroadcast)
[Nest] 187  - 06/05/2025, 1:45:49 AM     LOG [EventBus] Event handler registered [doc.created] (EventsListener.setDefaultPageOwner)
[Nest] 187  - 06/05/2025, 1:45:49 AM    WARN [CryptoHelper] AFFINE_PRO_PUBLIC_KEY is not set at compile time.
[Nest] 187  - 06/05/2025, 1:45:49 AM    WARN [CryptoHelper] AFFINE_PRO_LICENSE_AES_KEY is not set at compile time.
[Nest] 187  - 06/05/2025, 1:45:49 AM     LOG [JobHandlerScanner] Job handler registered [indexer.indexDoc] (IndexerJob.indexDoc)
[Nest] 187  - 06/05/2025, 1:45:49 AM     LOG [JobHandlerScanner] Job handler registered [indexer.deleteDoc] (IndexerJob.deleteDoc)
[Nest] 187  - 06/05/2025, 1:45:49 AM     LOG [JobHandlerScanner] Job handler registered [indexer.indexWorkspace] (IndexerJob.indexWorkspace)
[Nest] 187  - 06/05/2025, 1:45:49 AM     LOG [JobHandlerScanner] Job handler registered [indexer.deleteWorkspace] (IndexerJob.deleteWorkspace)
[Nest] 187  - 06/05/2025, 1:45:49 AM     LOG [JobHandlerScanner] Job handler registered [indexer.autoIndexWorkspaces] (IndexerJob.autoIndexWorkspaces)
[Nest] 187  - 06/05/2025, 1:45:49 AM   DEBUG [EventBus] Dispatch event: config.init (async)
[Nest] 187  - 06/05/2025, 1:45:49 AM VERBOSE [EventBus] Handle event [config.init] (OpentelemetryProvider.init)
[Nest] 187  - 06/05/2025, 1:45:49 AM VERBOSE [EventBus] Handle event [config.init] (URLHelper.init)
[Nest] 187  - 06/05/2025, 1:45:49 AM VERBOSE [EventBus] Handle event [config.init] (CryptoHelper.onConfigInit)
[Nest] 187  - 06/05/2025, 1:45:49 AM VERBOSE [EventBus] Handle event [config.init] (JobExecutor.onConfigInit)
[Nest] 187  - 06/05/2025, 1:45:49 AM VERBOSE [EventBus] Handle event [config.init] (ManticoresearchProvider.onConfigInit)
[Nest] 187  - 06/05/2025, 1:45:49 AM     LOG [SearchProviderFactory] Search provider [manticoresearch] registered.
[Nest] 187  - 06/05/2025, 1:45:49 AM VERBOSE [EventBus] Handle event [config.init] (ElasticsearchProvider.onConfigInit)
[Nest] 187  - 06/05/2025, 1:45:49 AM     LOG [RunCommand] Running RefreshFeatures0001...
[Nest] 187  - 06/05/2025, 1:45:49 AM VERBOSE [FeatureModel] Feature free_plan_v1 upserted
[Nest] 187  - 06/05/2025, 1:45:49 AM VERBOSE [FeatureModel] Feature pro_plan_v1 upserted
[Nest] 187  - 06/05/2025, 1:45:49 AM VERBOSE [FeatureModel] Feature lifetime_pro_plan_v1 upserted
[Nest] 187  - 06/05/2025, 1:45:49 AM VERBOSE [FeatureModel] Feature team_plan_v1 upserted
[Nest] 187  - 06/05/2025, 1:45:49 AM VERBOSE [FeatureModel] Feature early_access upserted
[Nest] 187  - 06/05/2025, 1:45:49 AM VERBOSE [FeatureModel] Feature unlimited_workspace upserted
[Nest] 187  - 06/05/2025, 1:45:49 AM VERBOSE [FeatureModel] Feature unlimited_copilot upserted
[Nest] 187  - 06/05/2025, 1:45:49 AM VERBOSE [FeatureModel] Feature ai_early_access upserted
[Nest] 187  - 06/05/2025, 1:45:49 AM VERBOSE [FeatureModel] Feature administrator upserted
[Nest] 187  - 06/05/2025, 1:45:49 AM     LOG [RunCommand] Running CreateIndexerTables1745211351719...
[Nest] 187  - 06/05/2025, 1:45:49 AM   ERROR [RunCommand] Failed to run data migration
[Nest] 187  - 06/05/2025, 1:45:49 AM   ERROR [RunCommand] TypeError: fetch failed
    at node:internal/deps/undici/undici:13510:13
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
    at async ManticoresearchProvider.createTable (file:///app/dist/main.js:121818:26)
    at async IndexerService.createTables (file:///app/dist/main.js:119862:13)
    at async Object.up (file:///app/dist/main.js:129980:9)
    at async RunCommand.runMigration (file:///app/dist/main.js:129522:13)
    at async RunCommand.run (file:///app/dist/main.js:129485:13)
    at async Command.<anonymous> (/app/node_modules/nest-commander/src/command-runner.service.js:170:24)
    at async Command.parseAsync (/app/node_modules/commander/lib/command.js:936:5)
    at async CommandRunnerService.run (/app/node_modules/nest-commander/src/command-runner.service.js:210:9) {
  [cause]: Error: connect ECONNREFUSED 127.0.0.1:9308
      at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1636:16)
      at TCPConnectWrap.callbackTrampoline (node:internal/async_hooks:130:17) {
    errno: -111,
    code: 'ECONNREFUSED',
    syscall: 'connect',
    address: '127.0.0.1',
    port: 9308
  }
}
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
node:internal/errors:983
  const err = new Error(message);
              ^

Error: Command failed: yarn predeploy
    at genericNodeError (node:internal/errors:983:15)
    at wrappedFn (node:internal/errors:537:14)
    at checkExecSyncError (node:child_process:882:11)
    at execSync (node:child_process:954:15)
    at runPredeployScript (file:///app/scripts/self-host-predeploy.js:43:3)
    at file:///app/scripts/self-host-predeploy.js:82:1
    at ModuleJob.run (node:internal/modules/esm/module_job:274:25)
    at async onImport.tracePromise.__proto__ (node:internal/modules/esm/loader:644:26)
    at async asyncRunEntryPointWithESMLoader (node:internal/modules/run_main:117:5) {
  status: 1,
  signal: null,
  output: [ null, null, null ],
  pid: 82,
  stdout: null,
  stderr: null
}

Node.js v22.16.0
fixing failed migrations.
running predeploy script.
yarn run v1.22.22
$ yarn prisma migrate deploy && yarn cli run
$ /app/node_modules/.bin/prisma migrate deploy
Prisma schema loaded from schema.prisma
Datasource "db": PostgreSQL database "affine", schema "public" at "affine-db:5432"

79 migrations found in prisma/migrations


No pending migrations to apply.
$ cross-env SERVER_FLAVOR=script node ./dist/main.js run

```


## `postgres:16-alpine` error:
```
Node.js v22.16.0
fixing failed migrations.
node:internal/errors:983
  const err = new Error(message);
              ^

Error: Command failed: yarn prisma migrate resolve --rolled-back 20250521083048_fix_workspace_embedding_chunk_primary_key
Error: Invariant violation: called markMigrationRolledBack on a database without migrations table.
   0: schema_core::state::MarkMigrationRolledBack
           with migration_name="20250521083048_fix_workspace_embedding_chunk_primary_key"
             at schema-engine/core/src/state.rs:481

error Command failed with exit code 1.

    at genericNodeError (node:internal/errors:983:15)
    at wrappedFn (node:internal/errors:537:14)
    at checkExecSyncError (node:child_process:882:11)
    at execSync (node:child_process:954:15)
    at fixFailedMigrations (file:///app/scripts/self-host-predeploy.js:57:7)
    at file:///app/scripts/self-host-predeploy.js:81:1
    at ModuleJob.run (node:internal/modules/esm/module_job:274:25)
    at async onImport.tracePromise.__proto__ (node:internal/modules/esm/loader:644:26)
    at async asyncRunEntryPointWithESMLoader (node:internal/modules/run_main:117:5) {
  status: 1,
  signal: null,
  output: [
    null,
    'yarn run v1.22.22\n' +
      '$ /app/node_modules/.bin/prisma migrate resolve --rolled-back 20250521083048_fix_workspace_embedding_chunk_primary_key\n' +
      'Prisma schema loaded from schema.prisma\n' +
      'Datasource "db": PostgreSQL database "affine", schema "public" at "affine-db:5432"\n' +
      'info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.\n',
    'Error: Invariant violation: called markMigrationRolledBack on a database without migrations table.\n' +
      '   0: schema_core::state::MarkMigrationRolledBack\n' +
      '           with migration_name="20250521083048_fix_workspace_embedding_chunk_primary_key"\n' +
      '             at schema-engine/core/src/state.rs:481\n' +
      '\n' +
      'error Command failed with exit code 1.\n'
  ],
  pid: 18,
  stdout: 'yarn run v1.22.22\n' +
    '$ /app/node_modules/.bin/prisma migrate resolve --rolled-back 20250521083048_fix_workspace_embedding_chunk_primary_key\n' +
    'Prisma schema loaded from schema.prisma\n' +
    'Datasource "db": PostgreSQL database "affine", schema "public" at "affine-db:5432"\n' +
    'info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.\n',
  stderr: 'Error: Invariant violation: called markMigrationRolledBack on a database without migrations table.\n' +
    '   0: schema_core::state::MarkMigrationRolledBack\n' +
    '           with migration_name="20250521083048_fix_workspace_embedding_chunk_primary_key"\n' +
    '             at schema-engine/core/src/state.rs:481\n' +
    '\n' +
    'error Command failed with exit code 1.\n'
}

Node.js v22.16.0

```

## `pgvector/pgvector:pg16` and `postgres:16-alpine` with `NODE_OPTIONS=--import=./scripts/register.js`

```
Node.js v22.16.0
node:internal/modules/esm/resolve:275
    throw new ERR_MODULE_NOT_FOUND(
          ^
Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/app/scripts/register.js' imported from /app/
    at finalizeResolution (node:internal/modules/esm/resolve:275:11)
    at moduleResolve (node:internal/modules/esm/resolve:860:10)
    at defaultResolve (node:internal/modules/esm/resolve:984:11)
    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:780:12)
    at #cachedDefaultResolve (node:internal/modules/esm/loader:704:25)
    at ModuleLoader.resolve (node:internal/modules/esm/loader:687:38)
    at ModuleLoader.getModuleJobForImport (node:internal/modules/esm/loader:305:38)
    at onImport.tracePromise.__proto__ (node:internal/modules/esm/loader:643:36)
    at TracingChannel.tracePromise (node:diagnostics_channel:344:14)
    at ModuleLoader.import (node:internal/modules/esm/loader:642:21) {
  code: 'ERR_MODULE_NOT_FOUND',
  url: 'file:///app/scripts/register.js'
}
```

--- a/packages/backend/server/scripts/self-host-predeploy.js
+++ b/packages/backend/server/scripts/self-host-predeploy.js
@@ -67,12 +67,18 @@ function fixFailedMigrations() {
         ) ||
         err.message.includes(
           'cannot be rolled back because it was never applied'
+        ) ||
+        err.message.includes(
+          'called markMigrationRolledBack on a database without migrations table'
         )
       ) {
         // migration has been rolled back, skip it
         continue;
       }
-      throw err;
+      // ignore other errors
+      console.log(
+        `migration [${migration}] rolled back failed. ${err.message}`
+      );
     }
   }
 }

